SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -156,6 +157,13 @@ void USpatialReceiver::OnAddComponent(const Worker_AddComponentOp& Op)
 			NetDriver->RegisterDormantEntityId(Op.entity_id);
 		}
 		return;
+	case SpatialConstants::VIRTUAL_WORKER_TRANSLATION_COMPONENT_ID:
+		if (NetDriver->VirtualWorkerTranslator != nullptr)
+		{
+			Schema_Object* ComponentObject = Schema_GetComponentDataFields(Op.data.schema_type);
```

Comment:
If ApplyVirtualWorkerManagerData arg can be const we could also make the pointer here const.

Reply:
It can't because it's also passed to the Worker SDK.